app/src/main/res/layout/pocket_onboarding.xml

```diff
@@ -49,7 +49,7 @@
         android:layout_width="88dp"
```

Comment:
Can you also match the other layout params of the Button? Mainly, here the widths and height are hard-coded, but they shouldn't be.